PhysicsTools/PatAlgos/python/slimming/modifiedJets_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
cfi files are supposed to have just one module
[there is no strict control over it though]